docs/articles/documentation/recipes/debugging-tests.md

```diff
@@ -48,6 +49,32 @@ You can also debug test code from an IDE.
 For instance, Visual Studio Code has a dedicated [TestCafe plugin](https://github.com/romanresh/vscode-testcafe).
 It allows you to run and debug your tests in the IDE using the context menu.
 
+## Debugging Test Behavior on the Client
+
+To debug test behavior on the client, use the [t.debug](../test-api/actions/debug.md) action.
+
+```js
+fixture `My fixture`
+    .page `https://devexpress.github.io/testcafe/example`;
+
+test('My test', async t => {
+    await t
+        .debug()
+        .click('#populate')
+        .click('#submit-button');
+});
+```
+
+When test execution reaches this action, it pauses so that you can open browser's developer tools
```

Comment:
Debugging does not stop on ClientFunctions and Selectors. I think we should add information.